dev-util/idea-ultimate/idea-ultimate-15.0.0.143.116.4_pre.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="0"
+PV_STRING="$(get_version_component_range 4-6)" # Always name EAP-versions with '_pre' for clarity!
+MY_PV="$(get_version_component_range 1-3)"
+MY_PN="idea"
+
+DESCRIPTION="A complete toolset for web, mobile and enterprise development"
+HOMEPAGE="http://www.jetbrains.com/idea"
+SRC_URI="https://download.jetbrains.com/idea/${MY_PN}IU-${PV_STRING}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal"
+IUSE=""
+KEYWORDS="~amd64 ~x86" # No keywords for EAP versions. Code quality sucks.
+
+DEPEND="!dev-util/${PN}:14
+	!dev-util/${PN}:15"
+RDEPEND="${DEPEND}
+	>=virtual/jdk-1.7"
+S="${WORKDIR}/${MY_PN}-IU-${PV_STRING}"
+
+QA_TEXTRELS="opt/${PN}-${MY_PV}/bin/libbreakgen.so"
+QA_PRESTRIPPED="opt/${PN}-${MY_PV}/lib/libpty/linux/x86/libpty.so
+	opt/${PN}-${MY_PV}/lib/libpty/linux/x86_64/libpty.so
+	opt/${PN}-${MY_PV}/bin/libyjpagent-linux.so
+	opt/${PN}-${MY_PV}/bin/libyjpagent-linux64.so"
+
+src_prepare() {
```

Comment:
don't want to reroll nitpicking, but why is this `rm -rf ...` instead of `rm -r ... || die`? This can cause silent bugs on version bumps